### ISSUE TITLE
Why we are down to 3 features only?

### ISSUE THREAD

**Issue by MohammedFCIS** [Dainanahan/IDG-DREAM-Drug-Kinase-Binding-Prediction-Challenge#3]

**Comment by alizat** [comment]
The remaining features are, for the most part, are not reproducible for test instances outside the training set. The features represent experimental measurements (other than pKd) for compounds (and compound-target pairs) that have been previously studied. Such measures are not available for cases that appear in the test set. Note that test instances involve experimental results that have been withheld for release until after the competition is over.

Furthermore, many of the remaining features that have been removed involve details pertaining to the literature where the experimental results were published (i.e. information that is not useful for training prediction models since this information cannot be replicated for test instances).
